home.html

```diff
@@ -1,322 +1,423 @@
-                </div>
-              </li>
-              <li class="listitem">
-                <div class="txtcontainer">
-                  <h3>Fameworks</h3>
-                </div>
-                <div class="imgcontainer">
-                  <img src="assets/collapse.svg" alt="" width="20" height="20" />
-                </div>
-              </li>
-              <li class="listitem">
-                <div class="txtcontainer">
-                  <h3>Skills</h3>
-                </div>
-                <div class="imgcontainer">
-                  <img src="assets/collapse.svg" alt="" width="20" height="20" />
-                </div>
-              </li>
-            </ul>
+                </li>
+                <li class="listitem">
+                  <div class="txtcontainer">
+                    <h3>Fameworks</h3>
+                  </div>
+                  <div class="imgcontainer">
+                    <img
+                      src="assets/collapse.svg"
+                      alt=""
+                      width="20"
+                      height="20"
+                    />
+                  </div>
+                </li>
+                <li class="listitem">
+                  <div class="txtcontainer">
+                    <h3>Skills</h3>
+                  </div>
+                  <div class="imgcontainer">
+                    <img
+                      src="assets/collapse.svg"
+                      alt=""
+                      width="20"
+                      height="20"
+                    />
+                  </div>
+                </li>
+              </ul>
+            </div>
           </div>
-        </div>
-      </section>
-
-    </main>
-  </div>
-</body>
-
-</html>
\ No newline at end of file
+        </section>
+        <section class="contact-container">
+          <form class="contact-form" action="https://formspree.io/f/xzbykgjg" method="POST">
+            <div class="contact-container-flex">
+              <h1 class="contact-me">Contact me</h1>
+              <p class="if-you">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d
+                love to help with it
+              </p>
+              <input
+                class="input-form username"
+                maxlength="30"
+                placeholder="Pick a name..."
+                type="text"
+                name="fullname"
+                required
+              />
+              <input
+                class="input-form email"
+                type="email"
+                placeholder="Email"
+                name="emailaddress"
+                required
+              />
+              <textarea
+                class="input-form textarea"
+                maxlength="500"
+                minlength="10"
+                name="message"
+                placeholder="Write your message here..."
+              ></textarea>
```

Comment:
- [ ] This Also should have a **`required`**, because it will be bad to send an empty message.